easier68k/core/opcodes/simhalt.py

```diff
@@ -114,9 +114,4 @@ def from_str(cls, command: str, parameters: str):
         :param command: The command itself (e.g. 'MOVE.B', 'LEA', etc.)
         :param parameters: The parameters after the command (such as the source and destination of a move)
         """
-        valid, issues = Simhalt.is_valid(command, parameters)
-        if not valid:
-            return None
-        # We can forego asserts in here because we've now confirmed this is valid assembly code
-
```

Comment:
Why the deletion?

Reply:
Because we started going off of the assumption that is_valid is being called outside of this method, so checking it again is just extra computation.